src/transformState.js

```diff
@@ -5,7 +5,27 @@
  * @param {Object[]} actions
  */
 function transformState(state, actions) {
-  // write code here
+  for (const action in actions) {
+    const point = actions[action];
+
+    if (point.type === 'addProperties') {
```

Comment:
```suggestion
  for (const action of actions) {
    if (action.type === 'addProperties') {
```

you can also use `for...of` to make your code simpler and to not create extra variable